src/Makefile

```diff
@@ -250,7 +250,9 @@ objcopy --add-gnu-debuglink $1.debug $1
 endef
 
 # Build the module...
-redisgraph.so: $(MODULE)
+libs: $(LIBS)
+objs: libs $(CC_OBJECTS)
+redisgraph.so: objs $(MODULE)
```

Comment:
both `objs` and `MODULE` contain `LIBS`

Reply:
Hi,

Well, I am not a Makefile expert, but I had to make these changes to make sure that `libs->objs->redisgraph.so` build order is followed in parallel builds. These depends on each other, so parallel builds fail because of missing prerequisites otherwise.

Does this cause any regression or something?